### ISSUE TITLE
Plural form of acronyms

### ISSUE THREAD

**Issue by lukasz-r** [jaraco/inflect#193]
The plural form of acronyms is formed by adding `S` to the acronym, e.g. `PPI` -> `PPIS`, however, the proper way is to add `s`, e.g. `PPI` -> `PPIs`:
https://editorsmanual.com/articles/plurals-of-abbreviations/

**Comment by jaraco** [comment]
I agree with your instinct. Also, thanks for providing a reference. Would you be interested and willing to work on an implementation?
